src/soilbiogeochem/SoilBiogeochemNitrogenStateType.F90

```diff
@@ -54,6 +67,11 @@ module SoilBiogeochemNitrogenStateType
      real(r8), pointer :: dyn_nh4bal_adjustments_col (:) ! (gN/m2) NH4 adjustments to each column made in this timestep via dynamic column adjustments (only makes sense at the column-level: meaningless if averaged to the gridcell-level)
      real(r8)          :: totvegcthresh                  ! threshold for total vegetation carbon to zero out decomposition pools
 
+     !Matrix-cn zgdu
+	 real(r8), pointer :: matrix_npools_vr_col (:,:,:)  !  col (gN/m3) vertically-resolved decomposing (litter, cwd, soil) N pools
+	 real(r8), pointer :: matrix_npools_col    (:,:)    ! (gN/m2)
```

Comment:
Add the comment that describes what this is here as well. It's in the line above. But, if someone adds something between the two it wouldn't have that context.

Reply:
These two variables were replaced by decomp_npools__vr_col and decomp_npools_col. So delete here